brightwind/analyse/analyse.py

```diff
@@ -434,7 +434,21 @@ def custom_agg(x):
         if i_dist == 0:
             distributions = distribution
         else:
-            distributions = pd.concat([distributions, distribution], axis=1)
+            # The below if statement is a workaround to be able to work with pandas >= 0.24.0, < 0.25.0, this should
+            # be replaced with distributions = pd.concat([distributions, distribution], axis=1)
+            # when these versions of pandas will not be supported anymore by brightwind library. This version of pandas
+            # doesn't allow to concatenate two pandas.Series with a CategoricalIndex if having a different index length
+            if len(distributions) >= len(distribution):
```

Comment:
@BiancaMorandi can you add a card so we will remember to depreciate this in the next breaking change version? Thanks.

Reply:
done #372